packages/bot-skeleton/src/utils/workspace.js

```diff
@@ -19,7 +19,7 @@ export const onWorkspaceResize = () => {
 
         const el_scratch_div = document.getElementById('scratch_div');
         if (el_scratch_div) {
-            el_scratch_div.style.width = '100vw';
+            el_scratch_div.style.width = 'calc(100vw - 3.2rem)';
```

Comment:
Adjusted the width of the workspace as per the new dasboard's tab content width.